### PR TITLE
PhpUnitNamespacedFixer - do not try to fix constant usage

### PR DESCRIPTION
**Pull request by kubawerlos**

### CODE REVIEW

File: tests/Fixer/PhpUnit/PhpUnitNamespacedFixerTest.php [hunk 1]
```diff
@@ -268,7 +275,7 @@ public static function provideClassIsFixedCases()
         $classmap = require __DIR__.'/../../../vendor/composer/autoload_classmap.php';
 
         foreach (array_keys($classmap) as $class) {
-            if (0 !== strpos($class, 'PHPUnit_')) {
+            if (0 !== mb_strpos($class, 'PHPUnit_')) {
```
Comment (keradus) [inline tests/Fixer/PhpUnit/PhpUnitNamespacedFixerTest.php:278]: do we have a case that we need `mb_` ?

Comment (kubawerlos) [inline tests/Fixer/PhpUnit/PhpUnitNamespacedFixerTest.php:278]:
```suggestion
            if (0 !== strpos($class, 'PHPUnit_')) {
```

Comment (kubawerlos) [reply tests/Fixer/PhpUnit/PhpUnitNamespacedFixerTest.php]: No, my bad, reverted.

**Review by GrahamCampbell — APPROVED** [review]

**Review by keradus — APPROVED** [review]

**Comment by keradus** [comment]
Thank you @kubawerlos.

**Pull request merged**